### DISCUSSION TITLE
Masking inquiry

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#458] [General]
If i have a specific file extension i dont want to be read or seen how would i format that as a mask? so in that directory im sharing and any sub folders in it and in that, etc can't see that specific file extension  *.ext

**Comment by rejetto** [comment]
```
"**/*.ext":
  can_read: false
```

that should be it, but test it.
i'm trying to have [a guide here](https://github.com/rejetto/hfs/wiki/Permissions).
can_read is the internal name for "can download".
this should be enough, as other permissions, like can_see, by default copy from can_read.
